app/src/main/java/de/danoeh/antennapod/config/StorageCallbacksImpl.java

```diff
@@ -125,6 +125,7 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     PodDBAdapter.KEY_CHAPTER_TYPE));
         }
         if(oldVersion <= 14) {
```

Comment:
Concerning the DB version, I can see your point. But we'll *have* to rev it if we wind up making changes between Alpha releases (which might be nice at some point, but we don't have that problem now).